cli/tests/04-add-page.bats

```diff
@@ -161,6 +161,23 @@ load helpers
   rm -r tmp
 }
 
+@test "Pages with camel-case path parameters generate code that compiles" {
```

Comment:
I diverged from the style of test description elsewhere in the file because it seemed like `elm-land add page.... compiles` wouldn't be clear on why that case might not compile and someone might remove the test as redundant.

Reply:
Test descriptions aren't set in stone, I like what you wrote here for the description!